staging/src/k8s.io/endpointslice/util/errors.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package endpointslice
+package util
```

Comment:
I wonder if we can factor this (followup) to avoid `package util` ?  Like, find a better home for all of these.